### DISCUSSION TITLE
Is tskit+tsinfer+tsdate applicable to sex chromosome?

### DISCUSSION THREAD

**Discussion by silvewheat** [tskit-dev/tskit#2174] [Q&A] [answered]
Hello,

Can I use tskit+tsinfer+tsdate in sex chromsome (X and Y)？

Best,
Yudong

**Comment by jeromekelleher** [comment]
Tskit is general and there is no concept of autosomes vs sex chromosomes. 

I'll defer to @hyanwong and @awohns for the details of using tsinfer and tsdate on sex chrs.

**Comment by hyanwong** [comment, marked as answer]
Hi Yudong: theoretically `tsinfer` should work fine on the X and Y chromosomes: it is fairly model-free. The X should be uncontroversial. For the Y, since you don't expect any recombination (apart from in the pseudoautosomal region), you might want to prioritise mismatching over recombination for the majority of the chromosome, which you could do with appropriate use of the mismatch parameter combined with a very low recombination rate (or a genetic map).

`tsdate` is more tricky. The prior assumes a neutral coalescent rate with an estimated $N_e$. The Ne will be different for X and Y than for the autosomes, but I think that as long as you account for this in the right way (e.g. I think the Ne for Y should be half that for the autosomes) then it should be more-or-less OK. Note that for the Y, there is likely to be selection on the entire haplotype which is likely to break assumptions of neutrality, so _caveat emptor_.
